scripts/sct_deepseg_gm.py

```diff
@@ -71,6 +80,35 @@ def run_main():
     out_fname = deepseg_gm.segment_file(input_filename, output_filename,
                                         model_name, int(verbose))
 
+
+    if '-qc' in arguments:
+        qc = os.path.abspath(arguments['-qc'])
+    if '-noqc' in arguments:
+        qc = False
+    if qc:
+        qc_path = qc
+        sct.printv('\nSave quality control images (in %s)...' % qc_path, verbose, 'normal')
+
+        import spinalcordtoolbox.reports.qc as qc
```

Comment:
Given that the QC generation seems to be the default behavior, consider moving these imports to the top.

Reply:
Totally agree. Just not now.